src/pfe/portal/modules/Project.js

```diff
@@ -246,15 +235,7 @@ module.exports = class Project {
     return metricsContextRoot;
   }
 
-  /**
-   * Function to return the full path to the project directory.
-   * @arg inPortal true if we want the location in the portal container (with username)
-   * @return the path to the project directory.
-   */
-  projectPath(_) {
```

Comment:
You could have left/updated the comment. Arguably we should be commenting our functions more not less. ;-)